xbmc/windowing/X11/WinSystemX11GLESContext.cpp

```diff
@@ -1,6 +1,6 @@
 #include "system.h"
 
-#if defined(HAVE_X11) && defined(HAS_EGL)
+#if defined(HAVE_X11) && defined(HAS_GLES)
```

Comment:
EGL is not GLES. builds for X11 require EGL. if this is not defines the build environment has an issue